custom_model_runner/datarobot_drum/__init__.py

```diff
@@ -0,0 +1 @@
+from .drum.custom_fit_wrapper import custom
```

Comment:
As you mentioned, it really looks weird, but maybe we could clean it a bit.

You are doing this to be able to do: `from datarobot_drum import custom`, right? So a couple of thoughts:

1. As I can recall, importing smth by default in __init__ is not a very good practice.
2. custom() method is not a very descriptive name; imagine how you are going to write documentation.

I'd suggest remove this import from here and:
Create a package and a module to be able to do: `from datarobot_drum.training_support.sklern my_magic_wrapper for estimator`
I think this could look nice and user friendly - we provide some helpers to make it easier to use drum and custom models in general.

Reply:
1. This is actually pythonic https://mikegrouchy.com/blog/be-pythonic-__init__py its pretty much the only thing that goes into the __init__.py file
2. I think we should defer naming to a third party. I think that requiring CFDSes in demos from writing that all out is pretty terrible UX, but I accept it could optimally be slightly more verbose than what I have. I passed what I have by Zach M and he said it looked good.

Reply:
Actually I think I'd like to keep it as `custom` because its short and sweet.

Reply:
1. Maybe here it's ok. I was referring to how it is done in datarobot-client, where __init__ in the root package is used to import anything, and another package can not be installed into the `datarobot` namespace.
2. custom() name doesn't make sense. It doesn't reflect any action what this function does. We are going to have it in examples - this will confuse CFDSs. Once demoed this will confuse customers. If we think no one reads our docs, we want our code do be self descriptive. That's why, going back to the item 1, I prefer imports to be done explicitly using absolute path.

Reply:
what about to_upload()